spacy_llm/tasks/ner.py

```diff
@@ -198,3 +204,21 @@ def scorer(
         examples: Iterable[Example],
     ) -> Dict[str, Any]:
         return get_ner_prf(examples)
+
+    def create_examples(
+        self,
+        examples: List[Example],
+    ) -> List[SpanExample]:
+        """Create NER examples from spaCy examples."""
```

Comment:
- Incomplete docstring.
- `create_examples(List[Example]) -> List[SpanExample]` reads a bit confusing to me due to the naming conflict between spaCy's `Example` and `spacy-llm`'s `Example` type(s). Perhaps we either rename this function or the `Example` types in `spacy-llm`?

Reply:
I'm in favor of the latter. `Demonstration` maybe? CC @svlandeg

Reply:
We don't have an actual `Example` type in `spacy-llm`, right? Only things like `RELExample` and `SpanExample`?

I do agree that `create_examples` is confusing here. Perhaps `create_prompt_examples`?

Reply:
Ah, that's right. `create_prompt_examples` is good with me.